### DISCUSSION TITLE
Document signing Sample

### DISCUSSION THREAD

**Discussion by witwaycorp** [MasterKale/SimpleWebAuthn#549] [Ideas]
I've been looking for a solution to use webauthn signed challenges for user attestation and your custom challenge section is the second write I've found on this topic. [See this link](https://developers.yubico.com/WebAuthn/Concepts/Using_WebAuthn_for_Signing.html?darkschemeovr=1). Would be great to see samples similar to what's on the post below on SimpleWebauthn and addressing the issue of replay attacks as you attached the custom signed challenge to a doc or email.

**Comment by MasterKale** [comment]
To be clear, you're asking for more of an example than what's proposed here in the docs? 

https://simplewebauthn.dev/docs/advanced/server/custom-challenges

**Reply by witwaycorp** [reply]
I guess it was more of a wish without an ask. Mostly wanted to link your project with that blog post suggestion. I'm working with a company right now to help me figure that out but they don't have the deep expertise. Appreciate the response.
